android/app/src/main/java/com/google/codeu/chatme/presenter/ProfilePresenter.java

```diff
@@ -157,7 +159,27 @@ public void onComplete(@NonNull Task<Void> task) {
      * Signs out current user
      */
     public void signOut() {
-        mAuth.signOut();
+        final DatabaseReference userRef = mRootRef.child("users").child(FirebaseUtil.getCurrentUserUid());
+
+        userRef.child("isOnline").setValue(Boolean.FALSE, new CompletionListener() {
+            @Override
+            public void onComplete(DatabaseError databaseError, DatabaseReference databaseReference) {
+                if (databaseError == null) {
+                    userRef.child("lastSeen").setValue(ServerValue.TIMESTAMP, new CompletionListener() {
```

Comment:
definitely helps with updating `lastSeen` when users log out instead of just when they shut the app/emulator down